Dynamic data structures/Advanced bracket balance/advancedBracketBalance.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+// The function checks whether the balance of parentheses is observed in the string
+bool advanceBracketBalance(const char* inputString);
```

Comment:
```suggestion
bool advancedBracketBalance(const char* inputString);
```
А лучше именовать функции глаголами в повелительной форме, то есть, например, `checkBracketsBalance`